installer/bundle/bundle_skel.sh

```diff
@@ -830,13 +830,18 @@ if [ "$installMode" = "I" -o "$installMode" = "U" ]; then
     fi
     curl_installed
     if [ $? -ne 0 ]; then
+        echo "curl was not found, attempting to install curl..."
+        install_extra_package curl
+    fi
+    check_program_in_path curl
```

Comment:
For the other packages we do this for (tar, sed), we also check the return value of the installed status like `sed_package_installed=$?`. For consistency and overall checking a `curl_package_installed=$?` like should be added here as well.